### DISCUSSION TITLE
从csredis切换redisfree后，使用过一次redis查询后，控制台无法关闭

### DISCUSSION THREAD

**Discussion by ghost** [2881099/FreeRedis#98] [General]
本地控制台dotnet run运行后，如果使用过一次redis连接，控制台无法关闭，我尝试释放了静态变量RedisClient才可以关闭。有什么好办法么

**Comment by ghost** [comment]
目前我在程序退出事件里释放了client临时解决了这个问题。

**Comment by guogangj** [comment]
这个问题能具体讲讲吗？我的情况是：dotnet 3.1程序在尝试关闭（ctrl+c）的时候会卡主90秒。在关闭时调用 Dispose方法也是一样。而新的dotnet 6的程序则没问题，但奇怪的是我将老的dotnet 3.1的程序升级到新的dotnet 6之后问题依旧，我对比不出哪里有区别。
